addons/website_event/views/event_event_views.xml

```diff
@@ -7,9 +7,9 @@
         <field name="priority" eval="5"/>
         <field name="inherit_id" ref="event.view_event_form"/>
         <field name="arch" type="xml">
-            <field name="company_id" position="after">
+            <xpath expr="//field[@name='company_id' and @groups='base.group_multi_company']" position="after">
```

Comment:
What about putting a name on its group (the main event form view has let and right groups before the notebook) ? That way we could target the company_id field that is inside "right_event_details" (or something like that) group ?

Reply:
Hello @tde-banana-odoo 

changes are done